diff --git a/android/src/main/java/com/bearblock/visioncameraocr/visioncameraocr/OcrFrameProcessorPluginRegistry.kt b/android/src/main/java/com/bearblock/visioncameraocr/visioncameraocr/OcrFrameProcessorPluginRegistry.kt
index c890ac7f6c631aa6270b67066ae949ac2ce9dbd3..09554944d1595a2e4de7d79a2201ce1196664349 100644
--- a/android/src/main/java/com/bearblock/visioncameraocr/visioncameraocr/OcrFrameProcessorPluginRegistry.kt
+++ b/android/src/main/java/com/bearblock/visioncameraocr/visioncameraocr/OcrFrameProcessorPluginRegistry.kt
@@ -9,8 +9,8 @@ class OcrFrameProcessorPluginRegistry {
      * This ensures the plugin is registered early in the app lifecycle.
      */
     init {
-      FrameProcessorPluginRegistry.addFrameProcessorPlugin("detectText") { proxy, options ->
-        OcrFrameProcessorPlugin(proxy, options)
+      FrameProcessorPluginRegistry.addFrameProcessorPlugin("detectText") { proxy ->
+        OcrFrameProcessorPlugin(proxy)
       }
     }
 
